TourMate/TourMate/Frontend/Plan/Views/AddPlanView/AddTransportView.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  AddTransportView.swift
+//  TourMate
+//
+//  Created by Tan Rui Quan on 10/4/22.
+//
+
+import SwiftUI
+
+struct AddTransportView: View {
```

Comment:
These are not used, the add and edit views

Reply:
They are present in the main currently though?

Reply:
it's in a separate folder